File: Iniciante/1007 - Diferença/JavaScript/1007.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split("\n");
  const [a, b, c, d] = lines.slice(0, 4).map(Number);
  const difference = a * b - c * d;
  console.log(`DIFERENCA = ${difference}`);
}

main();

File: Iniciante/1007 - Diferença/Python/1007.py
def main():
    A = int(input())
    B = int(input())
    C = int(input())
    D = int(input())

    print(f"DIFERENCA = {diferenca(A, B, C, D)}")


def diferenca(x, y, z, w):
    return x * y - z * w


main()
